app/controllers/offers_controller.rb

```diff
@@ -46,10 +41,19 @@ def index
       @offers = Offer.listable(current_user)
         .where.not(id: current_user.hidden_offers.pluck(:offer_id))
         .where.not(id: current_user.favored_offers.pluck(:offer_id))
-        .page(params[:page]).per(18)
+        .page(params[:page]).per(9)
     end
   end
 
+  def self.getFeatured(current_user)
```

Comment:
Ruby Style Guide recommends snake case for methods: 
```suggestion
  def self.get_featured(current_user)
```

Also, you'll need to make sure any **calling code is corrected**.